src/lib/pipeline.ml

```diff
@@ -316,16 +331,21 @@ let rec to_file_prefix:
       sprintf "%s-star-aligned" (to_file_prefix ?is sample)
     | Hisat (sample) ->
       sprintf "%s-hisat-aligned" (to_file_prefix ?is sample)
-    | Stringtie (sample) ->
-      sprintf "%s-stringtie" (to_file_prefix ?is sample)
+    | Stringtie (use_reference_gtf, sample) ->
+      sprintf "%s-%sstringtie"
+        (to_file_prefix ?is sample)
+        (match use_reference_gtf with
+        | `Yes -> "yesgtf"
```

Comment:
Hmm, but when would we actually be indifferent to how transcripts get assembled and named? 

I guess if you're checking to see whether StringTie runs at all and don't care about the output this might be the case but I'm not sure that's common enough to add these custom not-quite-boolean options.